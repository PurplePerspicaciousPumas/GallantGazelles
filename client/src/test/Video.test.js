import React from 'react';
import Video from '../components/Video.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Video />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});