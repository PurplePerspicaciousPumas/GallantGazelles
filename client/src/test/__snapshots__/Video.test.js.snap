// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <div>
    <iframe
      allowfullscreen={true}
      frameborder="0"
      height="360"
      mozallowfullscreen={true}
      src="https://player.vimeo.com/video/206456987"
      webkitallowfullscreen={true}
      width="640"
    />
  </div>
</section>
`;
